var/spack/repos/builtin/packages/evtgen/package.py

```diff
@@ -53,3 +53,6 @@ def build(self, spec, prefix):
         # due to libext_shared depending on lib_shared
         make('lib_shared')
         make('all')
+
+    def setup_run_environment(self, spack_env):
+        spack_env.set("EVTGEN", self.prefix.share)
```

Comment:
```suggestion
    def setup_run_environment(self, env):
        env.set("EVTGEN", self.prefix.share)
```
For consistency with other packages.